tests/unit/archive_base_test.py

```diff
@@ -207,7 +207,7 @@ async def test_download_file(mocker, tmp_path):
     def _ground_truth_download(url):
         return requests.get(url, timeout=1000).content
 
-    url = "https://github.com/catalyst-cooperative/pudl-archiver/blob/main/README.md"
+    url = "https://raw.githubusercontent.com/catalyst-cooperative/pudl-archiver/main/README.md"
```

Comment:
I'm not sure why the behavior on the old URL changed, but it seemed simpler to depend on the Markdown file itself rather than 6000 lines of HTML being dynamically generated by GitHub, and the test passes with the Markdown file so... 🤷🏼